### ISSUE TITLE
Should write type with queued child indentation

### ISSUE THREAD

**Issue by dsherret** [dsherret/ts-simple-ast#452]
See how this code does not indent properly:

```ts
sourceFile.addTypeAlias({
    isExported: true,
    name: "Test",
    type: writer => writer.write("string").newLine().write("| number")
});
```

**Comment by dsherret** [comment]
Marking this as breaking change because some people might be relying on this bug. I'll need to evaluate all the other places in the printers to see where it's not correctly using queued child indentation.

**Comment by dsherret** [comment]
Fixed in #461.
